js/lib/example.js

```diff
@@ -112,10 +114,11 @@ var BandsplotView = widgets.DOMWidgetView.extend({
         var bands = this.model.get('bands');
         var fdos = this.model.get('dos');
         var yLimit = this.model.get('energy_range');
+        var dosRange = this.model.get('dos_range');
         var showFermi = this.model.get('plot_fermilevel');
         var showLegend = this.model.get('show_legend');
 
-        bandPlot(that.uuidCanvas, that.uuidTextbox, bands, fdos, showFermi, showLegend, yLimit, ['#000000', '#de2d26', '#444444']);
+        bandPlot(that.uuidCanvas, that.uuidTextbox, bands, fdos, showFermi, showLegend, yLimit, dosRange ['#000000', '#de2d26', '#444444']);
```

Comment:
I guess you miss a comma after `dosRange`? Is that possible to test it in voila?

Reply:
Thanks for spoting this! I fixed it with the new commit.